app/javascript/src/components/Dashboard/Notes/Note/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Button, Avatar } from "neetoui";
+
+const Note = ({ notes }) =>
+  notes.map(note => (
+    <>
+      <div className="border-slate-300  shadow-slate-300 box-border h-40 w-full items-start rounded-sm border-2 bg-white p-4 shadow-md">
+        <div className="flex justify-between text-xl font-bold">
+          <Typography style="h5">{note.title}</Typography>
+          <Dropdown buttonStyle="text" icon={MenuVertical}>
+            <li>Edit</li>
+            <li>Delete</li>
+          </Dropdown>
```

Comment:
Go through https://github.com/bigbinary/neeto-ui/issues/1327#issuecomment-1253459534 and make relevant changes.

Reply:
![Screenshot 2022-09-22 at 12 01 25 PM](https://user-images.githubusercontent.com/71276756/191699303-cdc3bb35-7f9f-4a87-a550-ef743a99c65c.png)
![Screenshot 2022-09-22 at 12 01 50 PM](https://user-images.githubusercontent.com/71276756/191699333-00fb95a0-1db5-4fea-96dd-70c3112f844c.png)

Reply:
@nancy-binani Can you confirm that you are using neetoUI v4 itself? https://github.com/bigbinary/wheel/blob/master/package.json#L64

If not, please upgrade to v4 and lemme know if the error persists.

Refer: https://neetoui.onrender.com/?path=/docs/migration-guide-v3-to-v4--page

Reply:
I upgraded my version by running both the commands but the problem still persists.

Reply:
@nancy-binani Can you keep the destructuring of the `MenuItem` component outside the `Note` component? Like so:
```js
import {Dropdown, ...} from "neetoui";
const { Menu, MenuItem } = Dropdown;

const Note = () => {
...
};
```

Reply:
Will look into it.

Reply:
@nancy-binani Can you push the latest changes which is causing the error.

Reply:
I was replacing <li> tag with the following code. Then I was getting the error and I am unable to commit the code.My latest code before that is pushed.

Reply:
>Then I was getting the error and I am unable to commit the code

@nancy-binani What is preventing you from committing the code? Is it an eslint error? Then for the time being bypass the git hooks and push it. This is a one time exceptional case so as to debug the underlying issue. We should never push by bypassing git hooks.

Reply:
@nancy-binani I made the same changes and its working fine for me. I guess there is some problem in your neetoUI v4 update.
<img width="1009" alt="image" src="https://user-images.githubusercontent.com/43845165/191744625-2d7c092c-9067-4957-868c-8afc71f68ddb.png">
<img width="859" alt="image" src="https://user-images.githubusercontent.com/43845165/191744674-219a2b17-78db-40f2-ad3a-42596e6f5372.png">

Reply:
@nancy-binani can you confirm `@bigbinary/neetoui` version in package.json is `4.x`?
If not use this: 
yarn upgrade @bigbinary/neetoui@4 or
yarn yarn upgrade @bigbinary/neetoui --latest to update